tests/integration/targets/iosxr_bgp_neighbor_address_family/tests/common/merged.yaml

```diff
@@ -0,0 +1,116 @@
+---
+- debug:
+    msg: Start iosxr_bgp_nbr_address_family merged integration tests connection={{ ansible_connection}}
+
+- include_tasks: _remove_config.yaml
+
+- include_tasks: _populate_base_config.yaml
+
+- block:
+    - name: Merge the provided configuration with the exisiting running configuration
+      cisco.iosxr.iosxr_bgp_neighbor_address_family: &id001
+        config:
+          as_number: 65536
+          neighbors:
```

Comment:
Please add a test case (or update this one) to have a multiple neighbors and neighbors under VRF.

Reply:
Ack,made changes